howto/anbox/develop-platform.md

```diff
@@ -0,0 +1,84 @@
+Anbox Cloud provides a platform SDK which allows the development of custom platform plugins for the Anbox runtime.
```

Comment:
```suggestion
Anbox Cloud provides a platform SDK that allows the development of custom platform plugins for the Anbox runtime.
```

We should also link to https://anbox-cloud.io/docs/ref/platforms and give a short explanation and maybe some examples on why you would want to add your own platform.

Reply:
Added